docs/releases/1.7-NOTES.md

```diff
@@ -3,3 +3,10 @@ _This is a WIP document describing changes to the upcoming kops 1.7 release_
 # Significant changes
 
 * Default disk size increased to 64GB (masters) and 128GB (nodes).  This does have a higher cost, but also gives us more inodes & more iops (and more disk space, of course!)
+
+* Manifests are rewritten by default, which includes a normalization phase.
+  This can make it hard to understand the actual changes (as opposed to just the formatting changes).
+  A feature flag has been added, `export KOPS_FEATURE_FLAGS="-RewriteManifests"` which can be used
```

Comment:
Not sure what the purpose of this it actually is. To the uninitiated it looks like you are running an update on a cluster twice. Can we add more context or pull?